Week-1/aryan29/api.py

```diff
@@ -0,0 +1,110 @@
+import requests
+import bs4
+import sys
+from termcolor import colored,cprint
+import json
+with open("user_config.json","r") as f:
+    conf=json.load(f)
+    if(len(conf["mob"])==0):
+        cprint("Please Configure your user_config.json file and add Code of Atleast 1 mobile Phone to generate json",'red')
+        exit(0)
+class Api:
+    def get_data(self,mob):
+        b=requests.get(f"https://www.smartprix.com/mobiles/{mob[0]}")
+        l1=bs4.BeautifulSoup(b.text,'lxml')
+        price2=l1.select("#product-price .price")[0]
+        try:
+            a=requests.get(f"https://www.mysmartprice.com/mobile/{mob[1]}")
+            l=bs4.BeautifulSoup(a.text,'lxml')
+            price1=l.select(".prdct-dtl__prc-val")[0]
+            price=self.getminprice(price1,price2)
+        except:
+            price=price2
+        a=l1.select(".has-favorite-button , .cons span , .pros span")
+        f_list=a
+        f_list.append(price)
+        return f_list
+    def getminprice(self,price1,price2):
+        #print((price1.getText()),price2.getText()[1:])
+        if((price1.getText(),10)<(price2.getText()[1:],10)):
+            return price1
+        else:
+            return price2
+    def name_data(self,mob,code):
+        list=[]
```

Comment:
Why using a variable name the same as a builtin?

Now if I want to declare a list using the function list(), I'd have to use __builtins__.list in that scope. Not recommended right?